### ISSUE TITLE
Feature request: update to ethminer 0.11.0rc for 1060

### ISSUE THREAD

**Issue by Rapturoso** [nicehash/NiceHashMiner-Archived#1104]
Please update the NiceHashMiner to integrate support for the nVidia GTX 1060 optimised 0.11.0rc ethminer as this is a very welcomed 20% boost. Thank you.

**Comment by danmac** [comment]
Otherwise can we update ethminer ourselves? I only just started using NiceHash so not sure.

**Comment by darkmorpher** [comment]
[Genoil fork](https://github.com/Genoil/cpp-ethereum) deprecated and recommending https://github.com/ethereum-mining/ethminer

**Comment by Rapturoso** [comment]
Yes. I know the original Genoil version is depricated. Ethniner 0.011.0rc
is what I'm talking about! Ethminer has integrated this optimisation into
their prerelease version too. I have had no issues running it standalone
but I'd like to also take advantage of the NiceHash client automatically
switching profitability. As it stands, Ethminer's pre-release binary
(ethminer.exe) cannot be directly swapped with the older version you bundle
with NiceMash Miner 1.7.X.X as it just stop working as soon as it's called
from NiceHash. Can you integrate this ethminer 0.11.0rc into NiceHash so
that it does actually work please ? Thank you.

**Comment by tbitu** [comment]
This is important, there are many MH/s to be gained by implementing the new ethminer. Anyone looked at it?
